### PR TITLE
First draft of introduction

### PR DESCRIPTION
**Pull request by JohannesLandgraf**
<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/483"></a>

### CODE REVIEW

**Comment by Rajakavitha1** [comment]
@JohannesLandgraf  This is awesome!!!! May be we can do a bit of rephrasing for the titles. 
** What is GitPod**
**Cloud IDEs vs GitPod** ( This can may be a table with key differentiators) Users understand the distinction better when it is presented in a table versus paragraph.
**Advantages of GitPod**

File: src/routes/docs/beta/introduction.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+# Introduction to Gitpod
+
+[Gitpod](https://www.gitpod.io) is an open source platform for automated and **ready-to-code development environments** that blends into your existing workflow.
+
+It enables developers to describe their dev environment as code and **start configurable and fresh development environments for each new task** entirely in the cloud.
+
+<h2>Not another online IDE</h2>
+
+Today developers are emotionally attached (for better or worse) to their dev environments, give them names & massage them over time. They are like pets.
```
Comment (svenefftinge) [inline src/routes/docs/beta/introduction.md:13]: 'Dev Environments are treated like pets.'

File: src/routes/docs/beta/introduction.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+With Gitpod you treat dev environments as automated yet configurable resources you spin up when you need them and close down (and forget about) when you are done with your task. Dev environments become fully ephemeral. Only then you are **always ready-to-code** - immediately creative, immediately productive with the click of a button and without any friction.
```
Comment (svenefftinge) [inline src/routes/docs/beta/introduction.md:15]: 'With Gitpod dev environments are auztomated, yet configurable resources ...'

**Review by svenefftinge — APPROVED** [review]
👍 Let's merge?

**Pull request merged**